docs/sites.yml

```
@@ -6354,3 +6354,15 @@
   built_by: Criston Macarenhas
   built_by_url: https://github.com/cristonkrizz
   featured: false
+- title: The State of CSS Survey
+  main_url: https://stateofcss.com/
+  url: https://stateofcss.com/
+  source_url: https://github.com/StateOfJS/state-of-css-2019
+  description: >
+    Annual CSS survey, brother of The State of JS Survey.
+  categories:
+    - Survey
```

"Survey" isn't a category we currently support or want to add. Maybe Research is something that fits your needs as well.
```suggestion
    - Research
```

For a full list of categories please look at https://www.gatsbyjs.org/showcase/. We're going to cleanup categories which have only 1-3 sites.